mava/environment_loop.py

```diff
@@ -97,11 +100,9 @@ def _compute_episode_statistics(
     ) -> None:
         pass
 
-    def record_counts(self, episode_steps: int) -> counting.Counter:
+    def record_counts(self, episode_steps: int) -> Dict[str, counting.Number]:
```

Comment:
Is the type change here correct?

Reply:
Yup, `counter.increment()` returns a dict